packages/transport-native/package.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "trezor-transport-native",
+    "version": "0.1.0",
+    "description": "Trezor device transport layer for React Native",
+    "main": "lib/commonjs/index",
+    "module": "lib/module/index",
+    "types": "lib/typescript/index.d.ts",
+    "react-native": "src/index",
+    "source": "src/index",
+    "files": [
+        "src",
+        "lib",
+        "android",
+        "ios",
+        "cpp",
+        "trezor-transport-native.podspec",
+        "!lib/typescript/example",
+        "!android/build",
+        "!ios/build",
+        "!**/__tests__",
+        "!**/__fixtures__",
+        "!**/__mocks__"
+    ],
+    "scripts": {
+        "test": "jest",
+        "typescript": "tsc --noEmit",
+        "lint": "eslint \"**/*.{js,ts,tsx}\"",
+        "prepare": "bob build",
+        "release": "release-it",
+        "example": "yarn --cwd example",
+        "pods": "cd example && pod-install --quiet",
+        "bootstrap": "yarn example && yarn && yarn pods"
+    },
+    "keywords": [
+        "react-native",
+        "ios",
+        "android"
+    ],
+    "repository": "https://github.com/trezor/trezor-suite",
+    "author": "Trezor.io <daniel.suchy@satoshilabs.com> (https://github.com/trezor/trezor-suite)",
+    "license": "SEE LICENSE IN LICENSE.md",
+    "bugs": {
+        "url": "https://github.com/trezor/trezor-suite/issues"
+    },
+    "homepage": "https://github.com/trezor/trezor-suite#readme",
+    "publishConfig": {
+        "registry": "https://registry.npmjs.org/"
+    },
+    "devDependencies": {
+        "@types/react": "^16.9.19",
+        "@types/react-native": "0.62.13",
+        "jest": "^26.0.1",
+        "pod-install": "^0.1.0",
+        "prettier": "^2.0.5",
+        "react": "16.13.1",
+        "react-native": "0.63.4",
```

Comment:
`react`, `react-native` and relevant `@types` should have same version as in `@trezor/suite-native` package.json.
Or is there any reason to downgrade them?

Reply:
Good catch, I did not expect official RN package init to init with very old versions... Anyway I updated it to latest React Native because `suite-native` will be updated soon. Also pay attention that these are `devDependencies` package and also they are defined in `peerDependencies` with *, so it will use version by project where it is used